src/Websites/Websites/Websites.csproj

```diff
@@ -15,6 +15,21 @@
     <PackageReference Include="System.Security.Permissions" Version="4.5.0" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Don't this is needed?

Reply:
Removed it. It was added automatically by VS when changing the resource strings